install-review-tools.sh

```diff
@@ -2,8 +2,7 @@
 set -e
 HOME=/home/ubuntu
 
-# Use marco's ppa while we test the charm/charm-tools overlap stuff
-sudo add-apt-repository ppa:marcoceppi/charm-tools-2.0
+sudo add-apt-repository ppa:yellow/ppa
```

Comment:
@mbruzek it's not necessary. add-apt-repository can detect when you're not in an interactive shell and just assumes `-y` not bad to have though